src/llmcompressor/modifiers/transform/awq/base.py

```
@@ -267,16 +265,13 @@ def on_end(self, state: State, event: Event, **kwargs):
         self._assert_all_activations_consumed()
 
         # Recompute weight scales/zps for layers modified by smoothing
+        smoothed_layers = []
         for mapping in self._resolved_mappings:
             for layer in mapping.balance_layers + [mapping.smooth_layer]:
```

Ruff (0.15.12)

[warning] Consider `[*mapping.balance_layers, mapping.smooth_layer]` instead of concatenation

Replace with `[*mapping.balance_layers, mapping.smooth_layer]`

(RUF005)